### ISSUE TITLE
Diesel Engines and Modular Diesel Engines appear to dupe SU on SP world reload

### ISSUE THREAD

**Issue by pikminman13** [george8188625/Create-Diesel-Generators#77]
Hello, I've been extensively using Diesel Engines and Modular Diesel Engines, and have had an issue with SU duping. After some moderate testing, I've come to the following conclusions:

(This is for createdieselgeneratiors-1.19.2-1.2h, information collected via Stressometer)

Stress updates are the trigger for duping. values appear consistent on world load.
World closes appear to force a stress update sometimes.
Stress updates can be caused by both breaking stress users and clutches disabling users from getting it.
Adding a stress user to an unupdated system doubles the SU.
Breaking and replacing the stress user seems to additively increase SU as a function of the number of engines on the line.
Engines on separate networks do NOT impact each other.
Update duping only happens once per world load per network.
Stress consumption is unaffected and remains consistent to the number of users on the network.
Duping affects all SU generation on the network.
Adding modular engines to the line does NOT cause an update.
This affects both regular and modular engines.
The total SU generation is always the base number in the modification function, as with other generators on the network, their SU gets doubled/added depending as well.
Networks with no diesel engines are unaffected by the duping entirely, tested with various generators from both base Create and installed addons.

While not all of these hold true 100% of the time, they seem to hold at least 90% of the time, and regardless, there is absolutely a consistent issue with some amount of duping every time the world is loaded under a set of criteria. On my main singleplayer world, I gain around 300k SU per load with around 150k being the actual generated amount. I play "honorably" and respect the actual value of SU I should have to the best of my ability (obviously I will occasionally miss when I overstress the network but that doesn't really matter for the bug report).

**Comment by Saereth** [comment]
I've been able to indepdentently confirm these obersvations as well. Forge 43.3.8

**Comment by Yarden-zamir** [comment]
@Saereth are you on the new version? He mentioned it was fixed I think

**Comment by JulanDeAlb** [comment]
Yes it has been fixed but even after the update, the modular engine still dupes its SU. All the other Engines are fixed.

**Comment by Saereth** [comment]
yeah its the same

**Comment by Saereth** [comment]
The normal ones still dupe SU on world reload as well, the modular ones just do it worse because they introduce additional SU for additional motors but I suspect its all related.

**Comment by pikminman13** [comment]
![image](https://github.com/george8188625/Create-Diesel-Generators/assets/30882377/51d5e006-9fde-4155-9bdd-fe7ae8c8be9d)

**Comment by GustavJones** [comment]
Encountered the same issue on 1.20.1-1.2h version
